discord/guild.py

```diff
@@ -3100,14 +3099,32 @@ async def ban(
             ***Deprecated parameter***, same as ``delete_message_seconds`` but
             is used for days instead.
         reason: Optional[:class:`str`]
-            The reason the user got banned.
+            The reason the users got banned.
+
+        Returns
+        -------
+        Optional[List[List[:class:`Snowflake`], List[:class:`Snowflake`]]]
+            If banning a single member, returns nothing. Otherwise, returns two lists;
 
         Raises
         ------
+        ValueError
+            You tried to ban more than 200 users.
         Forbidden
             You do not have the proper permissions to ban.
         HTTPException
-            Banning failed.
+            No users were banned.
+
+        Example Usage: ::
+
+            # Ban a single member
+            await guild.ban(member, reason="Spam")
+
+            # Ban multiple members
+            successes, failures = await guild.ban(member1, member2, member3, ..., reason="Raid")
+
+            # Ban a list of members
+            successes, failures = await guild.ban(*members, reason="Hacked")
```

Comment:
This is formatted weirdly on docs

<img width="786" alt="image" src="https://github.com/Pycord-Development/pycord/assets/89910983/98976392-e135-4cb6-8038-33172024036a">

Reply:
Probably needs to be put above params n stuff